packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/rules/list/delete/routing.js

```diff
@@ -0,0 +1,43 @@
+import { TRACKING_HIT_PREFIX } from './constants';
+import { TRACKING_SUFFIX as RULES_TRACKING_SUFFIX } from '../../constants';
+import { TRACKING_SUFFIX as POLICIES_TRACKING_SUFFIX } from '../../../constants';
+import { TRACKING_SUFFIX as LISTENERS_TRACKING_SUFFIX } from '../../../../../constants';
+import { TRACKING_NAME } from '../../../../../../constants';
```

Comment:
Maybe it could be a good idea to gather these constants in a dedicated tracking constant file